include/d/actor/d_a_obj_hami4.h

```diff
@@ -1,17 +1,26 @@
 #ifndef D_A_OBJ_HAMI4_H
 #define D_A_OBJ_HAMI4_H
 
+#include "d/d_a_obj.h"
+#include "d/d_bg_w.h"
 #include "f_op/f_op_actor.h"
 
 class daObjHami4_c : public fopAc_ac_c {
 public:
+
+    enum Prm_e {
+        PRM_W = 0x08,
+        PRM_S = 0x00,
+    };
+
+
     inline cPhs_State _create();
     inline bool _delete();
     inline bool _draw();
     inline bool _execute();
-    void prm_get_swSave() const {}
+    int prm_get_swSave() const {return daObj::PrmAbstract<Prm_e>(this, PRM_W, PRM_S);}
```

Comment:
Let's rename the enum to have the field name in it
```suggestion
    int prm_get_swSave() const {return daObj::PrmAbstract<Prm_e>(this, PRM_SWSAVE_W, PRM_SWSAVE_S);}
```